### PR TITLE
update and remove of layers

### PR DESCRIPTION
**Pull request by michaelmattig**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

<TEXT>

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Layer Management im Manager](https://trello.com/c/CfIjdxgD/1733-layer-management-im-manager)

File: services/src/layers/layer.rs [hunk 1]
```diff
@@ -254,6 +272,18 @@ pub struct AddLayerCollection {
     pub properties: Vec<Property>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
+#[serde(rename_all = "camelCase")]
+// TODO: validate user input
```
Comment (ChristianBeilschmidt) [inline services/src/layers/layer.rs:277]: check it

File: services/src/pro/layers/postgres_layer_db.rs [hunk 2]
```diff
@@ -58,6 +58,64 @@ where
         Ok(layer_id)
     }
 
+    async fn update_layer(&self, id: &LayerId, layer: UpdateLayer) -> Result<()> {
```
Comment (ChristianBeilschmidt) [inline services/src/pro/layers/postgres_layer_db.rs:61]: add tests for the fns?

Comment (michaelmattig) [reply services/src/pro/layers/postgres_layer_db.rs:61]: they are tested through the API handlers.

File: services/src/layers/layer.rs [hunk 3]
```diff
@@ -65,6 +65,24 @@ pub struct AddLayer {
     pub metadata: HashMap<String, String>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
+// TODO: validate user input
```
[the remark this answers is not in the document]
Comment (michaelmattig) [reply services/src/layers/layer.rs]: 131db49ea68b62b3df27d14f617301b6917e3e2a

File: services/src/layers/layer.rs (hunk 1 above)
Comment (michaelmattig) [reply services/src/layers/layer.rs]: 131db49ea68b62b3df27d14f617301b6917e3e2a

File: services/src/pro/layers/postgres_layer_db.rs (hunk 2 above)
Comment (michaelmattig) [reply services/src/pro/layers/postgres_layer_db.rs:61]: they are tested through the API handlers.

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**